modules/apps/commerce/commerce-service/src/main/resources/com/liferay/commerce/internal/batch/04-object-definition.batch-engine-data.json

```diff
@@ -68,56 +68,28 @@
 					"system": true,
 					"type": "String"
 				},
-				{
-					"DBType": "String",
-					"businessType": "Text",
-					"externalReferenceCode": "RETURN_NOTE",
-					"indexed": true,
-					"indexedAsKeyword": false,
-					"label": {
-						"en_US": "Note"
-					},
-					"name": "note",
-					"required": false,
-					"state": false,
-					"system": true,
-					"type": "String"
-				},
-				{
-					"DBType": "String",
-					"businessType": "Picklist",
-					"defaultValue": "draft",
-					"externalReferenceCode": "RETURN_STATUS",
-					"indexed": true,
-					"indexedAsKeyword": false,
-					"label": {
-						"en_US": "Return Status"
-					},
-					"listTypeDefinitionExternalReferenceCode": "L_COMMERCE_RETURN_STATUSES",
-					"name": "returnStatus",
-					"required": true,
-					"state": true,
-					"system": true,
-					"type": "String"
-				},
 				{
 					"DBType": "String",
 					"businessType": "Aggregation",
-					"externalReferenceCode": "RETURN_REQUESTED_ITEMS",
+					"externalReferenceCode": "RETURN_CHANNEL_TOTAL_ACCEPTED",
 					"indexed": false,
 					"indexedAsKeyword": false,
 					"label": {
-						"en_US": "Requested Items"
+						"en_US": "Total Accepted"
 					},
-					"name": "requestedItems",
+					"name": "totalReceived",
```

Comment:
Mmm... we are summarizing different things (liter received + meters received). Are we sure?

Reply:
why the name is totalReceived if is total_accepted?